bundles/org.openhab.binding.solax/src/main/java/org/openhab/binding/solax/internal/SolaxBindingConstants.java

```diff
@@ -101,6 +105,14 @@ public class SolaxBindingConstants {
     public static final String CHANNEL_TODAY_FEED_IN_ENERGY = "today-feed-in-energy";
     public static final String CHANNEL_TODAY_CONSUMPTION = "today-consumption";
 
+    // Workmodes
+    public static final String WORKMODE_WAITING = "Waiting";
```

Comment:
Best would be to use an ENUM for these values. See the InverterType enum for example.

Could be even better to use these as I18N Keys constants so they get translated if they're used in the UI with a different locale, i.e. make the values all caps and translate them wherever you pass them to the UI/framework as states.

Reply:
This suggestion can wait for my next PR merge where I introduced Locale and time zone to the binding (from the framework. If you make the states an ENUM, I will later refactor them to use the translation files after my PR is merged...